tests/App/Admin/CategoryAdmin.php

```diff
@@ -20,6 +20,9 @@
 use Sonata\AdminBundle\Show\ShowMapper;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 
+/**
+ * @phpstan-extends AbstractAdmin<\Sonata\DoctrineORMAdminBundle\Tests\App\Entity\Category>
```

Comment:
isn't it beter to use a use?

Reply:
IDE like phpstorm are not understanding that the class will be used.
It could be confusing.